packages/babel-plugin-transform-block-scoping/README.md

```diff
@@ -81,3 +83,13 @@ for (let i = 0; i < 5; i++) {
 ```
 
 In extremely performance-sensitive code, this can be undesirable. If `"throwIfClosureRequired": true` is set, Babel throws when transforming these patterns instead of automatically adding an additional function.
+
+`tdz`
```

Comment:
Nit: can we set each option as a heading and add its type and default value ([example](https://github.com/babel/babel/tree/master/packages/babel-preset-react#development))

> ## Options
> 
> ### `throwIfClosureRequired `
> `boolean`, defaults to `false`.
> <...>
>
> ### `tdz`
> `boolean`, defaults to `false`.
> <...>